src/promptflow/promptflow/_sdk/_serving/utils.py

```diff
@@ -137,3 +142,48 @@ def encode_dict(data: dict) -> str:
     b64_data = base64.b64encode(zipped_data)
     # bytes -> str
     return b64_data.decode()
+
+
+def try_extract_trace_context(logger) -> Context:
```

Comment:
It seems that the open telemetry SDK has util functions that do the similar thing:
![image](https://github.com/microsoft/promptflow/assets/113209/f98537fc-9daa-4753-8707-0482edadbc29)

Reply:
Yes, this is using otel sdk,  otel doesn't provide existing util,  you need to write your code to extract the context based just like the sample code you got.